application/common/components/Emailer.php

```diff
@@ -30,6 +30,8 @@ class Emailer extends Component
     const SUBJECT_MFA_OPTION_REMOVED_DEFAULT = 'A 2-Step Verification option was removed from your %s account';
     const SUBJECT_MFA_ENABLED_DEFAULT = '2-Step Verification was enabled on your %s account';
     const SUBJECT_MFA_DISABLED_DEFAULT = '2-Step Verification was disabled on your %s account';
+    const SUBJECT_MFA_MANAGER_DEFAULT = 'Someone has sent you a login code for their %s account';
```

Comment:
Can we put the individual's name in here instead of `Someone`? That'll help the manager recognize it as importantI think

Reply:
Yes, I'll have to write some code to expand the ability to put variables into subject lines. I was considering it, but hadn't bit the bullet.